connecting-to-xdai/run-own-node.md

```diff
@@ -0,0 +1,116 @@
+# How to run your own xDai Node on Digital Ocean
+
+Welcome! cha0sg0d here. I'm a Dark Forest player and [dfdao](https://twitter.com/d_fdao) member. Here are the steps I took to sync an xDai node using the Nethermind client on a Digital Ocean instance.
+
+### Why run your own xDai Node?
+- No queue of other people's transactions
+- Faster data fetching (?)
+- Send more transactions, faster
+
+The options for xDai clients are [OpenEthereum](https://www.xdaichain.com/for-developers/install-xdai-client/parity) (formerly Parity) and [Nethermind](https://www.xdaichain.com/for-developers/install-xdai-client/nethermind). 
+
+### Why Nethermind
+
+from xDai [docs](https://www.xdaichain.com/for-developers/install-xdai-client/nethermind)
+> A majority of xDai validators are now running the Nethermind client. The latest version includes support for MEV via flashbots and increased default xDAI TxPool size from 1024 transactions to 2048.
+
+### Why Digital Ocean for hosting
+from Nethermind [docs](https://docs.nethermind.io/nethermind/first-steps-with-nethermind/hardware-requirements)
+> Generally the faster IO operations the better. For this reason Digital Ocean and UpCloud are so far the best cloud providers we have tested when running Nethermind nodes.
+
+### 1. Setting up a Digital Ocean Instance
+1. Buy a droplet (cloud computer). I recommend a machine with *at least* the following specs ($40/month):
+    * `Ubuntu 20.4 / 8 GB / 4 CPUs 160 GB SSD Disk 5 TB transfer`
+    * If this too slow, I synced a node in ~6 hours with the $80/month option
+      > If your sync is very slow (extended beyond two days) then very likely your setup cannot catch up with the chain progress.
+    * ![](../.gitbook/assets/xdai-node/droplet.png)
```

Comment:
Mention some kind of blurb on choosing password or ssh and link presumable a link to a relevant chapter they provide?
Maybe another blurb saying everything else should be fine as defaults

Reply:
I think I link to remote connect ssh elsewhere